tools/functional-tester/etcd-tester/member.go

```diff
@@ -125,7 +125,7 @@ func (m *member) SetHealthKeyV3() error {
 	}
 	defer cli.Close()
 	// give enough time-out in case expensive requests (range/delete) are pending
-	ctx, cancel := context.WithTimeout(context.Background(), 5*time.Second)
+	ctx, cancel := context.WithTimeout(context.Background(), 15*time.Second)
```

Comment:
timeout was increase to make sure that the etcd process on my local machine have enough time to process request due to  running mutiple etcd on a single machine might experiences slower i/o performance. I am unsure if this time constant matters much in production env.